### PR TITLE
[TASK] Add missing CustomSource implementation example

### PR DESCRIPTION
**Pull request by brotkrueml**
Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/347
Releases: main

### CODE REVIEW

File: Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst [hunk 1]
```diff
@@ -81,6 +81,32 @@ The corresponding event listener class:
         }
     }
 
+Example of a :php:`CustomSource` implementation:
+
+..  code-block:: php
```
Comment (linawolf) [inline Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst:86]: this should go into its own file

Comment (brotkrueml) [reply Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst:86]: Hmm, then we should move all event examples into separate files.

Comment (linawolf) [reply Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst:86]: In the long run I think that would be easier to keep up to date. But for now lets keep it like that

**Review by linawolf — APPROVED** [review]

File: Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst:86]: Yup, will put it on my list.

**Pull request merged**